test/macro_benchmark/example/memcpy_macro_benchmark.cpp

```diff
@@ -0,0 +1,32 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <algorithm>
+#include <cstring>
+
+#include <benchmark/benchmark.h>
+
+#include <seqan3/test/performance/units.hpp>
+
```

Comment:
Add a short discription. Something like:
`This test allocates 1Gb data with a string and copies it to another memory location.`